### PR TITLE
doc(github): add all the github markdown files

### PR DESCRIPTION
**Pull request by louis-bompart**

### CODE REVIEW

**Comment by louis-bompart** [comment]
- [ ] Figure out the contact email

**Review by olamothe — APPROVED** [review]

**Comment by louis-bompart** [comment]
Protip for review: 
image
use this button to preview the texts

**Comment by louis-bompart** [comment]
https://github.com/louis-bompart/special-octo-spork
Repo kinda setup to replicate what this PR will do to our repo

**Comment by louis-bompart** [comment]
@olamothe , ought to require another review, there are too many new files that I added after your review for it to be consider legit imho

**Comment by louis-bompart** [comment]
(was closed because of keyboard navigation oopsie)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - OS version: [e.g. Windows 10.0.19041]
+- Browser [e.g. Chrome, Safari]
+  - Browser version: [e.g. 90.0.4430.93]
+- Version of the CLI [e.g. 1.0.1]
+- Local Node version: [e.g. 14.16.0]
+- Local NPM/Yarn version: [e.g. 6.14.11]
+
+## Additional context
+
+Add any other context about the problem here.
```
Comment (jpmarceau) [inline .github/ISSUE_TEMPLATE/bug_report.md:44]:
```suggestion
Add any other contextual information about the problem here.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+5. See the error in the developer console/on the page
+
+## Expected behavior
+
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+
+If applicable, add screenshots to help explain your problem.
+Ensure that you do not display sensitive information such as token
```
Comment (jpmarceau) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```suggestion
Ensure that you do not display sensitive information such as tokens.
```

File: .github/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+- Other conduct which could reasonably be considered inappropriate in a
```
Comment (jpmarceau) [inline .github/CODE_OF_CONDUCT.md:38]:
```suggestion
- Any other kind of conduct which could reasonably be considered inappropriate in a
```

**Review by jpmarceau — APPROVED** [review]
👍

**Pull request merged**